pems-server/src/main/java/org/maroubra/pemsserver/monitoring/sensortag/SensortagSensor.java

```diff
@@ -49,13 +56,42 @@ public boolean start() {
                 startTemperatureCharacteristic() &&
                 startHumidityCharacteristic() &&
                 startBarometerCharacteristic() &&
-                startOpticalCharacteristic();
+                startOpticalCharacteristic() &&
+                startAccelerometerCharacteristic();
 
         if (!allCharacteristicsStarted) {
             sensortagDevice.disconnect();
             return false;
         }
+        return true;
+    }
 
+    /**
+     * Gets the specified characteristic on the Sensortag and disables it.
+     * @param sensor - sensor specified to be disabled
+     * @return successfully stopped the specified characteristic.
+     */
+    public boolean stopSelectedSensor(String sensor) {
+        switch(sensor) {
+            case "Thermometer":
```

Comment:
maybe it would look better if we can change "Thermometer" to public static final string THERMOMETER = "Thermometer" or some integer value and add it to sensortag class. then we can access it using SensortagSensor.THERMOMETER.

Reply:
yeah I completely agree, but lets not use integer values.